nodejs/week3/warm-up/index.js

```diff
@@ -0,0 +1,78 @@
+const express = require("express");
+const app = express();
+
+app.use(
+    express.urlencoded({
+        extended: true
+    })
+)
+app.use(express.json());
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.get("/calculator/multiply", (req, res) => {
+    if ("firstParam" in req.query && "secondParam" in req.query) {
+        const firstParam = [...req.query.firstParam];
+        const secondParam = [...req.query.secondParam]
+        const firstParaMult = firstParam.reduce((acc, currentValue) => parseFloat(acc) * parseFloat(currentValue));
+        const secondParamMult = secondParam.reduce((acc, currentValue) => parseFloat(acc) * parseFloat(currentValue));
+        const multiplication = parseFloat(firstParaMult) * parseFloat(secondParamMult)
```

Comment:
const values = Object.values(req.query).flat()
const multiplication = values.reduce((acc, val) => (acc) * (val))

and similar for every operation

Reply:
I used the spread operator as it accepts all the infinite paramters.

Reply:
/calculator/add?firstParam=12&secondParam=2 should return 14, and with code you wrote it will return 5 
but 12 + 2 = 14 isn't it? 🙂

Reply:
Ohh yes..I just saw it ..only the single digit is working fine. Thankyou so much Svetlana, for such a detailæ review, I really appreciate it. I will clean up the mess